tox.ini

```diff
@@ -6,6 +6,9 @@ envlist=
 [flake8]
 max-line-length= 100
 exclude= tests/*
+commands=
+	mypy --follow-imports=silent --warn-unused-ignores --ignore-missing-imports --no-strict-optional --check-untyped-defs --disallow-incomplete-defs --disallow-untyped-defs --disallow-any-generics -p eth_bloom`
```

Comment:
Since the library is so small, can we go directly with:

```python
mypy --strict --follow-imports=silent --ignore-missing-imports -p eth_bloom
```

Reply:
Yaa sure, will update it immediately. BTW why is Travis CI taking so long despite of the less number of test cases? Is it another issue with Travis?

Reply:
Two things:

1. The resources are shared across the entire ethereum organization with many projects kicking of CI tasks in parallel

2. TravisCI just doesn't seem to be very good (yet) on being fast. We are actively migrating away to CircleCI